CanonicalLtd/documentation-builder
# New styling and behaviour

Fixes #42.

QA
--

``` bash
pip3 install --user .
git clone -b latest git@github.com:nottrobin/maas-docs.git
bin/documentation-builder --base-directory maas-docs --output-path maas-docs/build --build-version-branches
xdg-open maas-docs/build/latest/en/index.html
```

Check styling looks sexy, and that the changes from #42 are there.
